github-actions/src/main/scala/org/typelevel/sbt/gha/WorkflowStep.scala

```diff
@@ -144,8 +146,18 @@ object WorkflowStep {
         env: Map[String, String] = Map(),
         params: Map[String, String] = Map(),
         timeoutMinutes: Option[Int] = None,
-        workingDirectory: Option[String] = None): Run =
-      Impl(commands, id, name, cond, env, params, timeoutMinutes, workingDirectory)
+        workingDirectory: Option[String] = None,
+        continueOnError: Boolean = false): Run =
```

Comment:
Same thing here: unfortunately, adding a default parameter (`= false`) will not fix bincompat. However, adding a deprecated overload with the old signature, that passes `continueOnError = false` to the new overload will work.